### ISSUE TITLE
Remote play error

### ISSUE THREAD

**Issue by vitalik2187836** [TheOfficialFloW/MiniVitaTV#8]
Every time i want to launch remote play session with minivitatv plugin installed i get C2-17025-5 or C2-15948-7 error.

**Comment by maikarusan** [comment]
yeah, ps3 remote play launches for me but the L2/R2/L3/R3 don't work.

**Comment by RitschiRitsch** [comment]
Issue still persistent. I'm getting the error C2-15948-7 when trying to remote play wit PS4 Link. However, in regular Vita mode without MiniVitaTV plugins loaded, PS4 Link works properly. Restoring remote play connection settings does not solve the issue.

**Comment by Abro61209** [comment]
Lol yep this has been an issue ever since this mediocre plugin released. What do I expect though, the rest of TheFlow’s work is pretty mediocre as well so doesn’t really surprise me. Just forget about this garbage plugin. I’ve noticed the only plugins that really work properly are the ones not made by TheFlow lol.
